Nested grouping and beta label removed from grouping (#3283)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

[[group-alerts]]
=== Group alerts

You can group alerts by rule name, user name, host name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.

> Select up to three fields for grouping alerts. The groups will nest in the order you selected them.

Author: ```suggestion
Select up to three fields for grouping alerts. The groups will nest in the order you selected them, and the nesting order is displayed above the table next to *Group alerts by*.
```

> Select up to three fields to create nested groups of alerts. The groups are nested in the order in which you select them.

Reviewer: ```suggestion
Select up to three fields by which to group alerts. The groups will nest in the order you selected them.
```

[role="screenshot"]
image::images/group-alerts.png[Alerts table with Group alerts by drop-down]

> Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group. The information displayed varies depending on the selected fields.

Reviewer 2: Can we maybe describe how the selection order will determine the nesting order, and that the order is displayed in the "Group alerts by" label?

Author: @stephmilovic We do mention the nesting order in the previous paragraph, above the screenshot, but I can add something about how the order is displayed in the label.
